src/components/TableOfContent.js

```diff
@@ -0,0 +1,24 @@
+export default function TableOfContent({ chapterTableOfContents, title }) {
+  if (!chapterTableOfContents?.length) return null;
```

Comment:
```suggestion
export default function TableOfContent({ chapterTableOfContents = [], title }) {
```
1. is there any reason why it has to return `null`?
2. `tableOfContents` should be enough.
3. Can you add input type with https://jsdoc.app/tags-type